src/ref.builds

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="*\ref\**\*.*proj">
+    <Project Include="*\ref\**\*.builds;*\ref\**\*.*proj">
```

Comment:
Yeah, but it'll noop, same as all the project-references to the ref projects that already come from the packages and src projects.  I didn't like this either, but seemed like the cleanest.  I tried to write the msbuild logic that'd exclude all projects under a builds folder but I couldn't do it without a task.  Alternatively we could add a builds file to every ref folder and change the convention.